love/src/redux/selectors/selectors.js

```diff
@@ -425,6 +427,20 @@ export const getLATISSState = (state) => {
   };
 };
 
+export const getAuthlistState = (state, subscriptions) => {
+  // const authlistData = getStreamsData(state, subscriptions);
+  // console.log(authlistData);
```

Comment:
If this code is not used, it should be removed

Reply:
I've removed this comment on further commits. Thanks!